backend/social_connections/telegram.py

```
@@ -0,0 +1,129 @@
+    return text[: limit - 1] + '…'
+
+
+def _mark_blocked(connection):
+    if connection is None:
+        return
+    connection.blocked_at = timezone.now()
+    connection.notifications_enabled = False
+    connection.save(update_fields=['blocked_at', 'notifications_enabled', 'updated_at'])
+
+
+def send_telegram_message(
```

Ruff (0.15.21)

[warning] Too many return statements (7 > 6)

(PLR0911)